### ISSUE TITLE
Blazor Azure ADB2C Authentication popup size does not match content

### ISSUE THREAD

**Issue by ahawes-clarity** [dotnet/aspnetcore#37365]
The default authentication popup when using Azure ADB2C Authentication does not account for additional "Sign in with social account" options.

![image](https://user-images.githubusercontent.com/6013584/136454171-b62e84fb-0d62-4e12-9c2b-99540236dca7.png)

I also was not able to find any documentation that would provide a way to control the size of the popup window.

**Comment by javiercn** [comment]
@ahawes-clarity thanks for contacting us.

This is something that MSAL exposes [(here)](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/51669adf5e2fe446bf94925fe77762b64e9513fc/lib/msal-browser/src/utils/PopupUtils.ts) but we don't in our wrappers. You can create your own `AuthenticationService.js` and configure the settings there directly.

**Comment by ahawes-clarity** [comment]
@javiercn thanks for the info. I was able to make a copy of the AuthenticationService.js file which was references in my index.html. I changed the reference to point to my copy and then changed the height value for the popup window. I realize that this is just a workaround and that if I update the version of MSAL I may need to redo this but it gives me what I need to solve the problem without too much work.

For others that might need to do this:

Here are my changes to index.html where WorkaroundAuthenticationService.js is a copy of AuthenticationService.js:
`    
    <script src="js/WorkaroundAuthenticationService.js"></script>`

I then updated WorkaroundAuthenticationService.js to set the popup height and width. The js file is minified so I had to do some searching and referencing of the TypeScript source to find what I needed.

Search for:
`ir="interaction_in_progress",sr="invalid_grant",ar=483,cr=600,ur=50,dr="msal.js.browser";`

It may be possible that yours differs based on what the minifier does but this should be enough info for you to find the code in your file.

In my case **ar** was width and **cr** was height.

Change the values as needed.
